tutorials/MD/prerun/run/em.log

```diff
@@ -1,22 +1,22 @@
-Log file opened on Sat Dec 14 15:07:00 2019
-Host: diablo  pid: 17682  rank ID: 0  number of ranks:  1
-                      :-) GROMACS - gmx mdrun, 2018.4 (-:
+                      :-) GROMACS - gmx mdrun, 2020.1 (-:
```

Comment:
Is there a point in storing the log files?

It's a very large file for github to track, maybe if we just want to make it available for those that can't use arcus we can zip the non-system (i.e. not a trajectory/structure/etc... file)?

Reply:
same thing for the trr file too (unless we plan on using it?)

Reply:
.log and .trr files deleted